cob_hardware_config/raw3-1/config/arm_cartesian_controller.yaml

```diff
@@ -0,0 +1,79 @@
+## Cartesian controller parameters
+chain_base_link: arm_base_link
+chain_tip_link: arm_ee_link
+root_frame: odom_combined
+
+# twist controller parameters
+twist_controller:
+  controller_interface: 0    #Velocity 0, Position 1, Trajectory 2, JointStates 3
```

Comment:
preferably you meld this file against any other cartesian_controller.yaml for consistency...